client_src/js/printerManagementRunner.js

```diff
@@ -1494,32 +1494,32 @@ function pluginListTemplate(plugin) {
                                 </div>
                                 <div class="meta">
                                     <small class="prop"><i class="fa fa-info"></i>&nbsp;<a target="_blank" href="${
-                                      plugin.page
-                                    }" title="${
+    plugin.page
```

Comment:
Hmm consider reverting these formatting changes for the plugins

Reply:
Hu? It's just the linter running.

Reply:
I've moved them back, seems eslint and prettier don't like these templates. Keep moving them to the first line.